org.elysium.parent/org.elysium/src/org/elysium/validation/LilyPondValidator.java

```diff
@@ -90,4 +90,16 @@ public void checkInclude(Include include) {
 			addIssue("Include with absolute location", getCurrentObject(), LilypondPackage.Literals.INCLUDE__IMPORT_URI, IssueCodes.ABSOLUTE_INCLUDE);
 		}
 	}
+
+	@Check
+	public void checkWindowsIncludeNormalized(Include include) {
+		if(LilyPondConstants.IS_WINDOWS && include.getImportURI()!=null) {
+			String includeString = include.getImportURI();
+			if(includeString.startsWith("file:/")) {
```

Comment:
Please extract `"file:/"` to a constant and use it in `LilyPondQuickfixProvider` also.